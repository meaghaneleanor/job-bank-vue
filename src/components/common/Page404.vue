<template>
  <div>
    <h1>We’re sorry, but we can’t find that page.</h1>
    <p>Please use the links below or the site search above.</p>
  </div>
</template>
