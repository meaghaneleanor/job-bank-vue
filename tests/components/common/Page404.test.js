import { shallowMount } from '@vue/test-utils';
import Page404 from '../../../src/components/common/Page404.vue';

const createComponent = shallowMount(Page404);

describe('Page404.vue', () => {
  describe('snapshot', () => {
    it('should display the expected html', () => {
      let wrapper = createComponent;
      expect(wrapper.html()).toMatchSnapshot();
    });
  });
});
