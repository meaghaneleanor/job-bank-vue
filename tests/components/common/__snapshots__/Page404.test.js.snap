// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page404.vue snapshot should display the expected html 1`] = `"<div><h1>We’re sorry, but we can’t find that page.</h1> <p>Please use the links below or the site search above.</p></div>"`;
